src/foam/java/Enum.js

```diff
@@ -1,106 +1,166 @@
-// Test with: foam.u2.ControllerMode.model_.javaSource();
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 
 foam.CLASS({
-  refines: 'foam.core.EnumModel',
+  package: 'foam.java',
+  name: 'EnumModel',
```

Comment:
This can just be called Enum, same way foam.java.Class is call Class not ClassModel.